xml/favicons.xml

```diff
@@ -2346,4 +2346,12 @@
     <param pos="0" name="service.product" value="R1Soft Server Backup Manager"/>
   </fingerprint>
 
+  <fingerprint pattern="^(?:eddb2b697c8c3dab5a9572de564976d1|b0f000f8041a6f1d2c366972b1dbad69)$">
+    <description>FatPipe Networks device web interface</description>
+    <example>eddb2b697c8c3dab5a9572de564976d1</example>
```

Comment:
Not a blocker - I wish we could add additional metadata here for what versions these are from, i.e. from your PR description you call out version `7.1.2r186p12` etc

that's something I would've found useful when I was adding new fingerprints for drupal - as I wasn't sure what versions were already fingerprinted when I was adding the fingerprints for newer versions

Reply:
That's a valid point. I've been in the habit of putting those details in the PR since there wasn't much in the way of documenting those as comments in the fingerprints. If we add that detail as comments to favicon fingerprints should we be adding them to all of the other fingerprints?

I'd like to consider even a future revision of the XML schema that includes this data.